backend/oasst_backend/scheduled_tasks.py

```diff
@@ -15,6 +15,8 @@
 from oasst_shared.utils import utcnow
 from sqlmodel import select
 
+# TODO: move each task to a separate file in the future to avoid cluttering this file
```

Comment:
I also don't think this should be a TODO, for now we have very few tasks so one file is file and probably will be for a long time

Reply:
Okay, I can remove that..